impeller/renderer/backend/vulkan/pipeline_library_vk.cc

```diff
@@ -128,6 +129,13 @@ static vk::UniqueRenderPass CreateRenderPass(const vk::Device& device,
     return {};
   }
 
+  // This pass is not used with the render pass. It is only necessary to tell
+  // Vulkan the expected render pass layout. The acutal pass will be created
```

Comment:
```suggestion
  // Vulkan the expected render pass layout. The actual pass will be created
```

Reply:
Its been a team effort on this sentence 😆